### PR TITLE
In Progress - Multiline Message In Dialog Box

### PR DESCRIPTION
**Pull request by LouisPi**
Scrolling needed. Also, would it be preferred if words weren't split up between lines when possible?

### CODE REVIEW

File: ui/dialog.py [hunk 1]
```diff
@@ -139,31 +139,36 @@ def refresh(self):
...
-        ctc = c.get_centered_text_bounds(self.el.message)
-        c.text(self.el.message, (ctc.left, 0))
-        c.text(self.displayed_label, (2, second_line_position))
+        choice_position = 10
+        chunk_y = 0
+        rendered_message = []
+        while self.el.message:
+                rendered_message.append(self.el.message[:self.el.o.cols])
+                self.el.message = self.el.message[self.el.o.cols:]
+                choice_position += 10
```
Comment (CRImier) [inline ui/dialog.py]: I suggest you make this hardcoded `10` into a separate GraphicalView attribute

**Comment by CRImier** [comment]
Better than the current state, but doesn't handle messages that are too long, of course. BTW, have you tried `format_for_screen` (in `ui/utils.py`) for text formatting?

File: ui/dialog.py (hunk 1 above)
Comment (LouisPi) [reply ui/dialog.py]: okay, will do

**Comment by LouisPi** [comment]
> Better than the current state, but doesn't handle messages that are too long, of course. BTW, have you tried `format_for_screen` (in `ui/utils.py`) for text formatting?

I looked into it but went with a similar method to the one Printer uses. I try out format_for_screen tonight.

**Comment by CRImier** [comment]
"too long" was about a message long enough to make the buttons not show => we'll have to implement scrolling eventually =)

**Comment by LouisPi** [comment]
> "too long" was about a message long enough to make the buttons not show => we'll have to implement scrolling eventually =)

Okay, thanks

File: ui/dialog.py (hunk 1 above)
Comment (LouisPi) [reply ui/dialog.py]: Or would o.char_height + 2 do the job?

**Comment by LouisPi** [comment]
Too long message error works - it raises the ValueError. Although I haven't fully tested it.

**Comment by CRImier** [comment]
Thank you, merged!

**Pull request merged**